DNN Platform/Library/Entities/Users/Profile/ProfilePropertyAccess.cs

```diff
@@ -243,7 +243,7 @@ public static string GetRichValue(ProfilePropertyDefinition property, string for
                         break;
                     case "richtext":
                         var objSecurity = new PortalSecurity();
-                        result = PropertyAccess.FormatString(objSecurity.InputFilter(HttpUtility.HtmlDecode(property.PropertyValue), PortalSecurity.FilterFlag.NoScripting), formatString);
+                        result = PropertyAccess.FormatString(objSecurity.InputFilter(property.PropertyValue, PortalSecurity.FilterFlag.NoScripting), formatString);

```

Comment:
this will cause the biohraphy show the html tags as content:
![20160330162204](https://cloud.githubusercontent.com/assets/3364448/14136093/9ebabaa2-f693-11e5-93fa-f17e03fcd091.png)